### DISCUSSION TITLE
19/68 security vendors flagged this file as malicious

### DISCUSSION THREAD

**Discussion by wiesercloud** [LorenEteval/Furious#127] [Q&A]
What do you say to the fact that VirusTotal defines the source code of the program as Win64:Malware-gen?

Proof: https://www.virustotal.com/gui/file/e1a5263d08d680075628869252166b53c5004d3f6a773765fe02d2ad399e0e07/detection

**Comment by LorenEteval** [comment]
Try figuring out what is source code and what is a release, then come back and ask questions. What do you say?
